be/server.go

```diff
@@ -12,11 +12,13 @@ import (
 const (
 	EndPointUser          = "/update_or_create_user"
 	EndPointReport        = "/report"
-	EndPointReadReport = "/read_report"
+	EndPointReadReport    = "/read_report"
 	EndPointGetMap        = "/get_map"
 	EndPointGetStats      = "/get_stats"
 	EndPointGetTeams      = "/get_teams"
 	EndPointPrivacyAndTOC = "/update_privacy_and_toc"
+	EndPointReadReferral  = "/readreferral"
+	EndPointWriteReferral = "/writereferral"
```

Comment:
In both cases comnvention so far is to use underscor: e.g. /read_referral

Reply:
Done.